core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/StopStartUpscaleCommissionViaCMHandler.java

```diff
@@ -93,10 +94,13 @@ protected Selectable doAccept(HandlerEvent<StopStartUpscaleCommissionViaCMReques
             Set<String> hostNames = allInstancesToCommission.stream().map(i -> i.getDiscoveryFQDN()).collect(Collectors.toSet());
             LOGGER.debug("HostNames to recommission: count={}, hostNames={}", hostNames.size(), hostNames);
 
-            HostGroup hostGroup = hostGroupService.getByClusterIdAndName(cluster.getId(), request.getHostGroupName())
-                    .orElseThrow(NotFoundException.notFound("hostgroup", request.getHostGroupName()));
+            Map<String, InstanceMetaData> hostsToRecommission = new HashMap<>();
```

Comment:
This is specifically not supposed to handle multiple hostGroups (at least for some time). Better to throw an error if there's more than one.